src/content/documentation/publish/self-distribution.md

```diff
@@ -61,6 +61,52 @@ When you choose to distribute an add-on yourself, they can be installed using th
   content: content
 %}
 
+{% capture content %}
+
+### Web download
+
+When distributing an add–on from your website, you have two options for making the signed add–on file available for download:
+
+- From a link on a webpage, like this:
+  ```html
+  <div id="example-option-1" class="install-ok">
+    <a href="https://addons.mozilla.org/firefox/downloads/file/3923300/facebook_container-2.3.2-fx.xpi">
+      Install my add-on
+    </a>
+  </div>
+  ```
+  This is the recommended approach.
```

Comment:
Do we want to move this up right below the chapter heading?

Reply:
That could make the presentation a bit awkward, I'd recommend leaving it here

Reply:
Can we make it more prominent, that this is the recommended approach? It doesn't seem very visible as standard text and it being at the end of the section, in the middle of a larger chapter.

Reply:
I understand this is a fairly soft recommendation and therefore doesn't particularly need to be highlighted @rpl any thoughts?

Reply:
I moved this sentence before the code snippet because the previous version made this sentence rather invisible.